src/els_methods.erl

```diff
@@ -94,10 +94,11 @@ initialize(Params, State) ->
    } = Params,
   InitOptions = maps:get(<<"initializationOptions">>, Params, #{}),
   ok = els_config:initialize(RootUri, Capabilities, InitOptions),
+  DbDir = application:get_env(erlang_ls, db_dir, default_db_dir()),
```

Comment:
This is to avoid test suites polluting the cache directory. The DB will be created in the local test dir for CT suites.